server/src/test/java/com/broadcom/lsp/cobol/positive/FolderTextRegistry.java

```diff
@@ -0,0 +1,77 @@
+package com.broadcom.lsp.cobol.positive;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.ListMultimap;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import com.google.inject.name.Named;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+
+/** Registry for Cobol source code files. */
+@Slf4j
+@Singleton
+public class FolderTextRegistry implements CobolTextRegistry {
+
+    private static final String POSITIVE_ENTRY = "positive";
+    private static final String NEGATIVE_ENTRY = "negative";
+    private static final String COPYBOOK_ENTRY = "copybooks";
+
+    private final ListMultimap<String, CobolText> texts = ArrayListMultimap.create();
+
+    @Inject
```

Comment:
I just noticed that this class is not used in the dependency injection, so it would be better to remove these `@Inject` and `@Named`

Reply:
Agree